site/en/docs/extensions/mv3/messaging/index.md

```diff
@@ -14,36 +14,34 @@ order to display a page action icon for that page.
 Communication between extensions and their content scripts works by using message passing. Either
 side can listen for messages sent from the other end, and respond on the same channel. A message can
 contain any valid JSON object (null, boolean, number, string, array, or object). There is a simple
-API for [one-time requests][1] and a more complex API that allows you to have [long-lived
-connections][2] for exchanging multiple messages with a shared context. It is also possible to send
-a message to another extension if you know its ID, which is covered in the [cross-extension
+API for [one-time requests][1] and a more complex API for [long-lived
+connections][2] to exchange multiple messages within shared context. It is also possible to send
+a message to another extension if you know its ID. That is covered in the [cross-extension
 messages][3] section.
 
 ## Simple one-time requests {: #simple }
 
 If you only need to send a single message to another part of your extension (and optionally get a
-response back), you should use the simplified [runtime.sendMessage][4] or [tabs.sendMessage][5].
-This lets you send a one-time JSON-serializable message from a content script to extension, or vice
-versa, respectively. An optional callback parameter allows you to handle the response from the other
-side, if there is one.
+response back), use the simplified [runtime.sendMessage()][4] or [tabs.sendMessage()][5].
+This lets you send a one-time JSON-serializable message from a content script to the extension, or vice
+versa. To handle the response, use the returned Promise. (For backward compatibility, you can still
+pass a callback as the last argument.)
 
 Sending a request from a content script looks like this:
 
 ```js
-chrome.runtime.sendMessage({greeting: "hello"}, function(response) {
-  console.log(response.farewell);
-});
+await response = chrome.runtime.sendMessage({greeting: "hello"});
+console.log(response.farewell);
 ```
 
-Sending a request from the extension to a content script looks very similar, except that you need to
+Sending a request from the extension to a content script is similar, except that you need to
 specify which tab to send it to. This example demonstrates sending a message to the content script
 in the selected tab.
 
 ```js
 chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
```

Comment:
```suggestion
chrome.tabs.query({active: true, lastFocusedWindow: true}, function(tabs) {
```

Let's update this to match the query parameters used in the [Tabs API example](https://developer.chrome.com/docs/extensions/reference/tabs/#get-the-current-tab).